src/samples/flow/netctl/Makefile

```diff
@@ -0,0 +1,34 @@
+sample-$(FLOW_NETCTL_SAMPLE) += \
+	list-network-services \
+	network-connect \
+	network-disconnect \
+	network-radios
+
+# list
+
+sample-list-network-services-$(FLOW_NETCTL_SAMPLE) := \
+	list-network-services.fbp
+
+sample-list-network-services-$(FLOW_NETCTL_SAMPLE)-deps := \
+	flow/netctl.mod
+
+#connection
+
+sample-network-connect-$(FLOW_NETCTL_SAMPLE) := \
+	network-connect.fbp
+
+sample-network-disconnect-$(FLOW_NETCTL_SAMPLE) := \
+	network-disconnect.fbp
+
+sample-network-connect-$(FLOW_NETCTL_SAMPLE)-deps := \
+	flow/netctl.mod
+
+#radios
+
+sample-network-radios-$(FLOW_NETCTL_SAMPLE) := \
+	network-radios.fbp
+
+sample-network-radios-$(FLOW_NETCTL_SAMPLE)-deps := \
```

Comment:
you do not need to repeat the `-deps := ` line for all samples. They will just reset. Move these up after `sample-$(FLOW_NETCTL_SAMPLE) += ...` line (before `# list` comment).